published_versions.json

```
@@ -196,6 +196,198 @@
+            "command": null
+          },
+          "accnone": {
+            "note": null,
+            "command": "conda install pytorch torchvision torchaudio cpuonly -c pytorch"
+          }
+        },
+        "libtorch": {
+          "accnone": {
+            "note": null,
+            "versions": {
+              "Download here (Pre-cxx11 ABI):": "https://download.pytorch.org/libtorch/cpu/libtorch-shared-with-deps-1.11.0%2Bcpu.zip",
+              "Download here (cxx11 ABI):": "https://download.pytorch.org/libtorch/cpu/libtorch-cxx11-abi-shared-with-deps-1.11.0%2Bcpu.zip"
+            }
+          },
+          "cuda10.2": {
+            "note": null,
+            "versions": {
+              "Download here (Pre-cxx11 ABI):": "https://download.pytorch.org/libtorch/cu102/libtorch-shared-with-deps-1.11.0%2Bcu102.zip",
+              "Download here (cxx11 ABI):": "https://download.pytorch.org/libtorch/cu102/libtorch-cxx11-abi-shared-with-deps-1.11.0%2Bcu102.zip"
+            }
+          },
+          "cuda11.x": {
+            "note": null,
+            "versions": {
+              "Download here (Pre-cxx11 ABI):": "https://download.pytorch.org/libtorch/cu113/libtorch-shared-with-deps-1.11.0%2Bcu113.zip",
+              "Download here (cxx11 ABI):": "https://download.pytorch.org/libtorch/cu113/libtorch-cxx11-abi-shared-with-deps-1.11.0%2Bcu113.zip"
+            }
+          },
+          "rocm4.x": {
+            "note": "LibTorch binaries are not available for ROCm, please build it from source",
+            "versions": null
+          }
+        }
+      },
+      "macos": {
+        "pip": {
+          "cuda10.2": {
+            "note": "# MacOS Binaries dont support CUDA, install from source if CUDA is needed",
+            "command": "pip3 install torch torchvision torchaudio"
+          },
+          "cuda11.x": {
+            "note": "# MacOS Binaries dont support CUDA, install from source if CUDA is needed",
+            "command": "pip3 install torch torchvision torchaudio"
+          },
+          "rocm4.x": {
+            "note": "<b>NOTE:</b> ROCm is not available on MacOS",
+            "command": null
+          },
+          "accnone": {
+            "note": null,
+            "command": "pip3 install torch torchvision torchaudio"
+          }
+        },
+        "conda": {
+          "cuda10.2": {
+            "note": "# MacOS Binaries dont support CUDA, install from source if CUDA is needed",
+            "command": "conda install pytorch torchvision torchaudio -c pytorch"
+          },
+          "cuda11.x": {
+            "note": "# MacOS Binaries dont support CUDA, install from source if CUDA is needed",
+            "command": "conda install pytorch torchvision torchaudio -c pytorch"
+          },
+          "rocm4.x": {
+            "note": "<b>NOTE:</b> ROCm is not available on MacOS",
+            "command": null
+          },
+          "accnone": {
+            "note": null,
```

@ezyang how about that?
```suggestion
            "note": "# MacOS Conda binaries are for x86_64 only, for M1 please use wheels",
```